x-pack/plugins/secops/server/lib/events/query.dsl.ts

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { createQueryFilterClauses } from '../../utils/build_query';
+import { FilterQuery } from '../types';
+import { EventsRequestOptions } from './types';
+
+export const EventFieldsMap = {
+  timestamp: '@timestamp',
+  'host.hostname': 'host.name',
+  'host.ip': 'host.ip',
+  'event.category': 'suricata.eve.alert.category',
+  'event.id': 'suricata.eve.flow_id',
+  'event.module': 'fileset.module',
+  'event.type': 'event.type',
+  'event.severity': 'suricata.eve.alert.severity',
+  'suricata.eve.flow_id': 'suricata.eve.flow_id',
+  'suricata.eve.proto': 'suricata.eve.proto',
+  'suricata.eve.alert.signature': 'suricata.eve.alert.signature',
+  'suricata.eve.alert.signature_id': 'suricata.eve.alert.signature_id',
+  'source.ip': 'source_ecs.ip',
+  'source.port': 'source_ecs.port',
+  'destination.ip': 'destination.ip',
+  'destination.port': 'destination.port',
+  'geo.region_name': 'destination.geo.region_name',
+  'geo.country_iso_code': 'destination.geo.country_iso_code',
+};
```

Comment:
Did you test with the latest ECS loaded on the new test server? It looks like you copied and then changed the values back to the older Infra server's version of ECS we loaded?

From above these fields should be:
```ts
  'event.module': 'event.module',
  'source.ip': 'source.ip',
  'source.port': 'source.port',
```
And they look like the older version. Or did I get them wrong when I did a commit the other day? .. or something else ... 😕

Reply:
Let me check

Reply:
ok that's fix